packages/components/src/scroll-to/index.js

```diff
@@ -0,0 +1,33 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { Component, createRef } from '@wordpress/element';
+
+class ScrollTo extends Component {
+	constructor() {
+		super();
+		this.scrollTo = this.scrollTo.bind( this );
+	}
+
+	componentDidMount() {
+		setTimeout( this.scrollTo, 250 );
+	}
+
+	scrollTo() {
+		if ( this.ref.current && this.ref.current.offsetTop ) {
+			window.scrollTo( 0, this.ref.current.offsetTop );
+		} else {
+			setTimeout( this.scrollTo, 250 );
+		}
+	}
+
+	render() {
+		this.ref = createRef();
+		return (
+			<span ref={ this.ref }></span>
```

Comment:
You'll want to add some buffer to take into account the navbar. If you shrink the window veritcally, you can see the top of the section hidden under the navbar.

![Screen Shot 2019-08-23 at 12 49 35 PM](https://user-images.githubusercontent.com/1922453/63559256-b005f680-c5a4-11e9-8e04-c7c466f159cf.png)

Reply:
@psealock Thanks, good catch. I added an offset parameter in https://github.com/woocommerce/woocommerce-admin/pull/2799/commits/f0e7b2e80c20239b3f49b69b2768c15fbddea73b